web-console/future-openshift/operators-overview/operators-overview.md

```diff
@@ -3,16 +3,15 @@
 This documentation details how operators will be surfaced on the Status page. Rather than exposing CSV objects, the status page should now expose the CRs being managed by identified CSVs in designated groups.
 
 ## Group by: Application
-![image](img/operators8-app.png)
+![image](img/groupby-application.png)
 * In the group by Application view, operator managed CRs will be grouped together with the CSV object name as the group header (AMQ Streams).
```

Comment:
Since applications are based on a label and users have control over what they put in the application...what if they have an application that includes multiple Operators/CRs? Maybe we need a way to scale the header to include multiple things?

Reply:
Hm, that's a really good question. Would you expect to see something like 3scale / AMQ Streams for example? If I recall correctly I think we wanted to automatically add the label app and sort the resources that way (although I remember the conversation around grouping was fairly extensive so I could be wrong), but I hadn't thought more about if a user edits them. At that point I assume the header would just reflect the new label, and if users wanted to indicate that multiple operators are included they'd need to use something like app=3scale-amq for example. Curious if @beanh66 remembers more about this?

Reply:
I do think it will work very nice in a lot of cases, but also that anyone at any time could apply label of "app=xyz" to whatever they'd like. So if a customer bundles a few things together into one application like 3scale (View Operator) / AMQ Streams (View Operator) we might want to have some flexibility with the design scale.

Reply:
Would it be too weird to just have a heading per app instance (xyz, 3scale, AMQ Streams) and then show the same resource under multiple headings if it has more than one app associated with it? That would mean the same resource would appear multiple times in the overall list.

Or alternatively, treat any combination of apps as a new heading, "xyz; 3scale" but then would need to have a "View operator details" link per app, if it's an operator?

Reply:
I chatted a bit with someone from the OLM team and it sounds like users shouldn't / wouldn't really be overriding the operator labels. My understanding is that each resource can only have one "app=____" label, and that would default to the operator. I could be mistaken but it sounds like any changes the user makes to that would just get changed back, so this might not be an issue. I think it's probably good to merge now and we can follow up again if needed!